modules/java/native/java-parser.c

```diff
@@ -32,7 +32,7 @@ static CfgLexerKeyword java_keywords[] =
   { "java",        KW_JAVA },
   { "class_path",  KW_CLASS_PATH},
   { "class_name",  KW_CLASS_NAME},
-  { "option",      KW_OPTION},
+  { "options",     KW_OPTIONS},
```

Comment:
This breaks the user interface. Introducing `options` is great, but we must support the old syntax, too. See: `KWS_OBSOLETE`.

Reply:
@alltilla : okay. I have added KWS_OBSOLETE for option.